.changeset/giant-pillows-lick.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/table': patch
```

Comment:
Suggestion: going off of the i18n tickets, should these changesets be a minor bc youre adding props?

Reply:
Proooooooooooooooooooooooooobably 🙃